cesium/internal/index/domain.go

```diff
@@ -29,23 +29,35 @@ type Domain struct {
 var _ Index = (*Domain)(nil)
 
 // Distance implements Index.
-func (i *Domain) Distance(ctx context.Context, tr telem.TimeRange, continuous bool) (approx DistanceApproximation, err error) {
+func (i *Domain) Distance(
+	ctx context.Context,
+	tr telem.TimeRange,
+	continuous bool,
+) (approx DistanceApproximation, bounds DomainBounds, err error) {
```

Comment:
very minor - ignore this if you don't think it's necessary - `domainBounds` may be a better name since there are too many things in this method that can be called `bounds`.

Reply:
Good idea